src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +52,28 @@
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService
+                = (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService =
+                (AuthenticationService) injector.getInstance(AuthenticationService.class);
+        try {
+            authenticationService.register("121212@gmail.com", "qwerty");
+        } catch (RegistrationException e) {
+            throw new RuntimeException(e);
+        }
+        User user;
+        try {
+            user = authenticationService.login("121212@gmail.com", "qwerty");
+        } catch (AuthenticationException e) {
+            throw new RuntimeException(e);
+        }
+        System.out.println(user);
```

Comment:
Ensure that sensitive information, such as passwords, is not exposed when printing the `User` object. Consider overriding the `toString()` method in the `User` class to exclude sensitive data.